.github/setup.py.tmpl

```diff
@@ -0,0 +1,36 @@
+from setuptools import setup
+
+
+with open("README.md", "r") as fh:
+    long_description = fh.read()
+
+setup(
+    name='pyuniqid',
+    packages=['pyuniqid'],
+    version='${VERSION}',
+    license='MIT',
+    description='A Unique Hexatridecimal ID generator',
+    long_description=long_description,
+    long_description_content_type="text/markdown",
+    author='Boris Skurikhin',
+    author_email='boriskurikhin@gmail.com',
+    url='https://github.com/boriskurikhin/pyuniqid',
+    download_url='${DOWNLOAD_URL}',
+    keywords=[
+        'unique',
+        'id',
+        'unique identifier',
+        'hexatridecimal',
+        'unique id',
+        'uniqid'],
+    install_requires=[
+        'netifaces',
+        'numpy'
+    ],
+    classifiers=[
+        "Programming Language :: Python :: 3",
+        "License :: OSI Approved :: MIT License",
+        "Operating System :: OS Independent",
+    ],
+    python_requires='>=3.5',
+)
```

Comment:
Template `setup.py` used for envsubst. You can see the tokens `${VERSION}` and `${DOWNLOAD_URL}` where those environment variables will be placed.